sentry-spring-boot-starter-jakarta/src/test/kotlin/io/sentry/spring/boot/jakarta/SentryAutoConfigurationTest.kt

```diff
@@ -683,6 +684,47 @@ class SentryAutoConfigurationTest {
             }
     }
 
+    @Test
+    fun `when OpenTelemetryLinkErrorEventProcessor is on the classpath and auto init off, creates OpenTelemetryLinkErrorEventProcessor`() {
```

Comment:
So, for this to work one would have to set:
- `sentry.auto-init=false` in the properties AND
- set the `SENTRY_AUTO_INIT` env var to `false`

Correct?

Reply:
This is spring magic that translates the `SENTRY_AUTO_INIT` env var into a `sentry.auto-init` property. So you don't set both, you only set the env var.

Reply:
For the test i simply provided the property. Could look into env var, no sure that's possible tho.